pandas/tests/arrays/test_integer.py

```diff
@@ -98,9 +98,9 @@ def test_repr_array_long():
     data = integer_array([1, 2, None] * 1000)
     expected = (
         "<IntegerArray>\n"
-        "[ 1,  2, NA,  1,  2, NA,  1,  2, NA,  1,\n"
-        " ...\n"
-        " NA,  1,  2, NA,  1,  2, NA,  1,  2, NA]\n"
+        "[ 1,  2, NA,  1,  2, NA,  1,  2, NA,  1,\n "
+        "...\n "
```

Comment:
It feels like in this case the previous option is more natural. Since every line represents the content of a line if this was the content of a file.

So, for example, if we have a file:
```
XX
 X
```
It probably makes more sense to have:
```python
data = (
    "XX\n"
    " X\n"
)
```
Than
```python
data = (
    "XX\n "
    "X\n"
)
```
What do you think?

Reply:
Hmm I do get you're point.

so if I can check if the line is ending with a ```\n```, it's OK for the next line to begin with a space? right?

correct me if I misunderstood

Reply:
Yes, I think the rule could be that if one line finishes with `\n`, we don't really care what's in the next, it should always be ok.

But if you think it makes sense, you can revert these few lines for now, so we can merge this. And then we can see in more detail in the other PR (but I think it should just be that condition).

Reply:
> But if you think it makes sense, you can revert these few lines for now, so we can merge this. And then we can see in more detail in the other PR (but I think it should just be that condition).

Sure!

Reply:
> Yes, I think the rule could be that if one line finishes with `\n`, we don't really care what's in the next, it should always be ok.

Logic for this test case is now implemented at https://github.com/pandas-dev/pandas/pull/30755/commits/a4ec49ee2022319eaac9602be61d4e40eabf105e